### PR TITLE
API-8746: Replace markdown with html

### PR DESCRIPTION
**Pull request by gileswells**
### Description

https://vajira.max.gov/browse/API-8746

The purpose of this PR is to change the auth, consumer, and provider docs over to using HTML / React Components instead of using markdown. The goal here is to be able to remove some of the hacky workarounds needed to allow the auth docs to function as designed.

Originally this work was to include updating the markdown loader to a maintained package but that's been delayed till the dev portal updates to using webpack v5.

### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [ ] Accessibility concerns have been considered and addressed
  
  - [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

File: src/components/oauthDocs/TestUsers.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const TestUsers = (): JSX.Element => (
+  <>
+    <SectionHeaderWrapper heading="Test Users" id="test-users" />
+    <p>
+      Some APIs require test users and test data. Most of the test data provided from the Lighthouse
+      platform comes from internal VA systems, is not real data, and are reset based upon new
+      recordings of underlying services. We provide{' '}
+      <a href="https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts.md">
+        test accounts
```
Comment (wwanamaker) [inline src/components/oauthDocs/TestUsers.tsx:12]: I think you're missing a space after "test accounts" here. 

image

File: src/components/oauthDocs/IdToken.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      Your application must validate JWT signatures. This allows your application to verify that the
+      provided JWT originates from our authorization servers and prevents your application from
+      accepting a JWT with claims that are attempting to impersonate one of your users.
+    </p>
+    <h3>Header</h3>
+    <p>
+      The JWT&quot;s header has two fields, <code>alg</code> and <code>kid</code>. <code>alg</code>{' '}
+      indicates the algorithm that was used to sign the JWT, and <code>kid</code> identifies the key
+      that was used to sign the JWT. Signing keys and associated metadata are accessible from
+      <a href="https://sandbox-api.va.gov/oauth2/.well-known/openid-configuration">
```
Comment (wwanamaker) [inline src/components/oauthDocs/IdToken.tsx:25]: Missing a space before this link: 
image

**Review by imcgonigle — DISMISSED** [review]
This looks good to me, but I would consider using absolute paths instead of relative paths to make things slightly easier to read/understand.

instead of
```javascript
import { AuthCodeFlowContentProps } from '../../containers/documentation/AuthorizationDocs';
import { APISelector } from '../apiSelector/APISelector';
```

do

```javascript
import { AuthCodeFlowContentProps } from 'containers/documentation/AuthorizationDocs';
import { APISelector } from 'components/apiSelector/APISelector';
```

**Comment by gileswells** [comment]
@imcgonigle The issue with that is when it comes to unit testing. The absolute pathing doesn't match up so it causes issues / fails the tests because it cannot find the imports.

**Review by imcgonigle — APPROVED** [review]

**Pull request merged**